v1/ao/context.go

```diff
@@ -19,6 +23,14 @@ func newSpanContext(ctx context.Context, l Span) context.Context {
 	return context.WithValue(ctx, contextSpanKey, l)
 }
 
+func FromXTraceIDContext(ctx context.Context, xTraceID string) context.Context {
+	aoCtx, err := reporter.NewContextFromMetadataString(xTraceID)
+	if err != nil {
+		fmt.Printf("FAILED WITH from ID ctx %+v \n", err)
```

Comment:
Can you print it with a levelled logging like `log.Debug`?

Reply:
Just creating the PR so i can documented the changes :) It contains a lot of debug/experimental which will be removed later on